pkg/knuu/instance.go

```diff
@@ -58,6 +30,38 @@ type Instance struct {
 	jaegerEndpoint           string
```

Comment:
please place jaegerEndpoint with other jaeger namings close to each other for readability. 
In addition, please make comment lines between each options

Reply:
https://github.com/celestiaorg/knuu/pull/174/commits/943f4bebc89cf114a5059e693fbb825751faaddd